### ISSUE TITLE
"..." button is not brought in foreground when it is selected using the "Tab" key

### ISSUE THREAD

**Issue by Softvision-VladBacia** [mozilla/activity-stream#1993]
**[Affected versions]:**
-Firefox 45.0 and up
-Activity Stream Dev 1.3.0-dev-1484695187.

**[Affected Platforms]:**
-All Windows

**[Prerequisites]:**
-Latest Activity Stream add-on version (1.3.0-dev) was previously installed on a profile.
-Have more history so that the "Highlights" area is populated with visited websites.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites.
2. Open a New Tab so that Activity Stream page is displayed.
3. Drag&Drop one of the Highlights tiles.
4. Press "Tab" key from keyboard.

**[Expected result]:**
-"..." button is brought in foreground and it is selected.

**[Actual result]:**
-"..." button is selected but is hidden.

**[Notes]:**
- [Here](https://drive.google.com/open?id=0B6QogV2AJFKiY0tuT2dGaGNteVU) is a video of the issue.
- If the user cycles through Highlight tiles using the "Tab" key, then each selected Highlight tile will have a text cursor displayed in the left part of the favicon. (can be seen in the video)
- If the context menu is display, it cannot be closed with the "Esc" key.
- These scenarios are reproducible for Top Sites and Highlights.
- Related with #1978 .

**Comment by Softvision-VladBacia** [comment]
This issue has been verified on A-S 1.14.0-dev-1497551877-9d19af4 and I cannot reproduce is. Thus marking this as Verified-Fixed.
